frontend/components/tags/_tags.scss

```diff
@@ -1,3 +1,8 @@
+.govuk-tag--small {
```

Comment:
I've added this because `govuk-tag` does not support smaller font sizes (via a modifier or font token) and since we're using a smaller 14px font size for the content here the default 16px tag text looks too large.